File: back/src/main/java/com/example/back/controllers/dto/TeamDto.java
package com.example.back.controllers.dto;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@NoArgsConstructor
@Getter
@Setter
public class TeamDto {
    private String name;
    private Long id;
    private Boolean isFavorite;
}

File: front/amadeusSports/src/app/dto/TeamDto.ts
export class TeamDto {
    id?: number;
    name: string = '';
    isFavorite: boolean = false
    constructor(id: number, name: string, isFavorite: boolean) {
        this.id = id;
        this.name = name;
        this.isFavorite = isFavorite;
    }
}